CorsixTH/Lua/dialogs/resizables/file_browsers/save_game.lua

```diff
@@ -63,6 +63,14 @@ function UISaveGame:abortName()
   self.new_savegame_textbox.panel:setLabel(_S.save_game_window.new_save_game)
 end
 
+--! Updates the textbox to selected file
+--!param node (table) Selected file
+function UISaveGame:updateTextbox(node)
+  local name = string.gsub(node,".sav","")
```

Comment:
This is the source of your `Autoe` bug as you strip sav from the name. Either way you don't want to be doing this on directories or invalid nodes.